account_payment_unece/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+
+{
+    'name': 'Account Payment UNECE',
+    'version': '10.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'license': 'AGPL-3',
+    'summary': 'UNECE nomenclature for the payment methods',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
+    'depends': ['account_payment_mode', 'base_unece'],
```

Comment:
@alexis-via the dependency to `account_payment_mode` is here, so the comment is not correct.